backend/fms/settings.py

```diff
@@ -208,13 +208,10 @@
     },
 }
 
-
-SESSION_EXPIRE_AT_BROWSER_CLOSE = False
-SESSION_COOKIE_AGE = 60 * 60  # 1 hour
-
+# Mechanism in order to automatically logout the user after 4 hours
 SIMPLE_JWT = {
-'ACCESS_TOKEN_LIFETIME': timedelta(seconds=20),
-'REFRESH_TOKEN_LIFETIME': timedelta(seconds=20),
+'ACCESS_TOKEN_LIFETIME': timedelta(hours=4),
+'REFRESH_TOKEN_LIFETIME': timedelta(minutes=5),
```

Comment:
Not sure what this does, but is 5 minutes really a good delay?

Reply:
the refresh token is used to get a new access token. even though it is usually longer-lived than the access token, for our purpose (expiring the session in the back-end) i think it should not be a long time...  if the browser is closed, we dont want to have a long refresh token lifetime.. (as we discussed, the front-end expiration is handled by the timer)

Reply:
But won't that have the effect of logging the user out 10 minutes after they close their browser tab?